src/Reducers/initialState.js

```diff
@@ -1,6 +1,19 @@
 import { initialState as viewRequests } from '../State/ViewRequests/reducers';
 
 const initialState = {
+  currentUser: {
```

Comment:
Could we add a `// TODO Replace Stub Data with Real Data` comment (or something like that) anytime we stub data.   That way we can search our code base and find loose ends down the line.

Reply:
You got it dude!